### PR TITLE
Add initial code to display the browse categories

### PR DESCRIPTION
**Pull request by skywalkerdude**
Add initial code to display the browse categories. Unable to test on device/simulator yet, can only test with live preview

### CODE REVIEW

File: Hymns/Browse/BrowseCategoriesViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import Combine
+import Foundation
```
Comment (finben) [inline Hymns/Browse/BrowseCategoriesViewModel.swift:2]: I don't think you need Foundation here

File: Hymns/Browse/CategoryViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Foundation
```
Comment (finben) [inline Hymns/Browse/CategoryViewModel.swift:1]: I don't think you need Foundation here

File: Hymns/Repository/CategoriesRepository.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import Combine
+import Foundation
```
Comment (finben) [inline Hymns/Repository/CategoriesRepository.swift:2]: I don't think you need Foundation here

File: Hymns/Browse/CategoryViewModel.swift (hunk 2 above)
Comment (skywalkerdude) [reply Hymns/Browse/CategoryViewModel.swift:1]: we need it for `ObservableObject`

**Pull request merged**